src/App/components/Editors/EditPopover.tsx

```diff
@@ -16,14 +16,13 @@ export const EditRow = styled(Flex).attrs({
   alignItems: "center",
 })``;
 
-interface PopoverProps {
+export interface EditPopoverProps extends PositionProps, SpaceProps {
```

Comment:
Should you use `BoxProps` here like you did in `ColorPicker.tsx` instead of `SpaceProps`? Also in `TextLocationToggle.tsx`

Reply:
Ahh yes, after finding and using `BoxProps` for the color picker at the end, I didn't re-examine where else I could have used it. Thanks for spotting that - I've fixed it now.